src/components/TutorialCarouselDialog/components/tutorial0.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Typography, withStyles } from 'material-ui';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import styles from '../styles';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+export default class Tutorial0 extends Component {
```

Comment:
since we're not using `state` here, can we please make this a stateless component